server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -66,6 +66,30 @@ export function create(
 		);
 	});
 
+	/**
+	 * Signs a tenant token using the given tenant's key.
+	 */
+	router.post("/tenants/:id/accesstoken", (request, response) => {
+		const tenantId = request.params.id;
+		const scopes = request.body.scopes;
+		const documentId = request.body.documentId ?? "";
+		const user = request.body.user;
+		const lifetime = request.body.lifetime;
+		const ver = request.body.ver;
+		const jti = request.body.jti;
```

Comment:
Stylistic nit, feel free to ignore :). Could also include `documentId` and do the coalescing when passing it to `signToken()` below.
```suggestion
		const { scopes, user, lifetime, ver, jti } = request.body;
		const documentId = request.body.documentId ?? "";
```